allennlp/semparse/worlds/wikitables_variable_free_world.py

```diff
@@ -0,0 +1,174 @@
+"""
+We store the information related to execution of context sensitive execution of logical forms here.
+We assume that the logical forms are written in the variable-free language described in the paper
+'Memory Augmented Policy Optimization for Program Synthesis with Generalization' by Liang et al.
+The language is the main difference between this class and `WikiTablesWorld`. Also, this class defines
+an executor for the variable-free logical forms.
```

Comment:
I think the executor code should be in a different file.

Also, an idea that Chen suggested that we don't have to do now, but would be nice if we could do eventually: when you're defining the language in the type declaration, also implement the execution of each of those functions, and have it all in one place.  I think there's a clean way to do this, but let's worry about that later.

Reply:
Hmm, actually, having the execution depend on the context makes doing it that way pretty hard...  Still, it'd be nice to think about how to make the language-definition process easier.

Reply:
I agree there are cleaner ways to do this. For example, it would be nice if we only defined the executor, and the type declaration could be inferred from it automatically.